README.md

```diff
@@ -73,9 +128,12 @@ The nimut/testing-framework ships database fixtures for several TYPO3 CMS core d
 - sys_language
 - tt_content
 
-To use the database fixtures you can trigger an import in your test file
+Override `$testExtensionsToLoad` in your test class to create extension tables in the test database,
+and trigger an import:
 
```

Comment:
Why did you remove the line above? This is incorrect and not what is explained with

$this->importDataSet('ntf://Database/pages.xml');

Reply:
While trying to load data fixtures it was not enaugh to to add ``$this->importDataSet('ntf://Database/pages.xml');``, we noticed that variable `$testExtensionsToLoad` was required too to make it work.

Reply:
This is independent from a database import. $testExtensionsToLoad is required to execute your functional tests anyways. Otherwise your extension isn't accessible for the tests.

See: https://wiki.typo3.org/Functional_testing#How_to_write_functional_test.3F and https://de.slideshare.net/cpsitgmbh/functional-tests-with-typo3

Reply:
The instructions in a README are meant to be self-contained. Given we agree on that, can we list all essential details in it? A sequence of instructions should allow to reproduce the expected result. Assumptions should be mentioned clearly, right before the start of the sequence.

Where should the paragraph mentioning to "override `$testExtensionsToLoad` in your test class" go instead?

Reply:
Oh, just noticed that this has been discussed in the past in issue #28. PR #29, upon your request, is waiting to be merged since then. Please, go ahead if it's fine. Thank you! :+1:

Reply:
The framework is meant to run your extension tests against multiple versions. It is based on the functionality of TYPO3 CMS 6.2 and 7.6 that was extracted to the typo3/testing-framework. Unfortunately with that extraction it isn't possible to test against multiple versions. It is not scope of that framework to explain how testing in extension with TYPO3 CMS works in general. We cannot offer those documentation and ask everyone to look in the already provided documentations of the community (see #43 ). As the $testExtensionsToLoad property is an important property to be able to run your tests with any of the both testing-frameworks, it is not related to nimut/testing-framework and not mentioned in this documentation. Otherwise we need to mention all important properties (which we cannot offer).

I really hope you understand that the functionality of the test cases and the concept of testing offered by this framework are extracted from already existing sources. And we cannot provide any support within the framework itself.